spectator-api/src/main/java/com/netflix/spectator/api/Registry.java

```diff
@@ -52,6 +52,28 @@
   Id createId(String name, Iterable<Tag> tags);
 
   /**
+   * Creates a dynamic identifier for a meter. All ids passed into other calls should be created by the registry.
+   *
+   * @param name
+   *     Description of the measurement that is being collected.
+   * @return
+   *     The newly created dynamic identifier.
+   */
+  DynamicId createDynamicId(String name);
+
+  /**
+   * Creates a dynamic identifier for a meter. All ids passed into other calls should be created by the registry.
+   *
+   * @param name
+   *     Description of the measurement that is being collected.
+   * @param tagFactories
+   *     Other factories that can generate other dimensions that can be used to classify the measurement.
+   * @return
+   *     The newly created dynamic identifier.
+   */
+  DynamicId createDynamicId(String name, Iterable<TagFactory> tagFactories);
```

Comment:
In this case I think it is fine for it to break the api since we aren't at 1.x yet and currently all the registry implementations are local to this repo. However, I am concerned that the check in the build didn't flag the issue:

https://github.com/Netflix/spectator/blob/master/spectator-api/build.gradle#L11

I do see that it ran as part of the travis build.